packages/angular_devkit/build_optimizer/src/transforms/wrap-enums_spec.ts

```diff
@@ -14,6 +14,7 @@ import { getWrapEnumsTransformer } from './wrap-enums';
 const transform = (content: string) => transformJavascript(
   { content, getTransforms: [getWrapEnumsTransformer] }).content;
 
+// tslint:disable-next-line:no-big-function
```

Comment:
note, we have some effort from google3 to teach tslint how to have relaxed constraints in tests, would be worth pushing that forward and using it...